src/Dropzone/assets/src/style.css

```diff
@@ -1,72 +1,130 @@
+:root {
+  --dropzone-background: white;
+  --dropzone-background-hover: #dddddd;
+  --dropzone-border-color: #aaaaaa;
+  --dropzone-border-color-hover: #666666;
+  --dropzone-text-color: ##333333;
+
+  --dropzone-spacing: 8px;
+  --dropzone-radius: 8px;
+
+  --dropzone-width: cacl(100% - 2 * var(--dropzone-spacing));
+  --dropzone-height: 120px;
+  --dropzone-image-size: 100px;
+
+  --dropzone-file-svg: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path fill="none" stroke="currentColor" stroke-linejoin="round" stroke-width="2" d="M10 3v4a1 1 0 0 1-1 1H5m14-4v16a1 1 0 0 1-1 1H6a1 1 0 0 1-1-1V7.914a1 1 0 0 1 .293-.707l3.914-3.914A1 1 0 0 1 9.914 3H18a1 1 0 0 1 1 1Z"/></svg>');
+  --dropzone-close-svg: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24"><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="m15 9l-6 6m0-6l6 6m6-3a9 9 0 1 1-18 0a9 9 0 0 1 18 0"/></svg>');
```

Comment:
I think this would be better in the theme, not sure everyone want this

Reply:
Using SVGs as data URLs (data:image/svg+xml;base64,...) is often restricted in enterprise environments. This is mainly due to security policies like CSP (Content Security Policy) that block data: schemes or inline content. Encoded SVGs can also contain JavaScript or CSS, which raises concerns about XSS vulnerabilities. Additionally, data URLs bypass HTTP-level caching and CDN optimization, making them less efficient for large-scale or public-facing applications.

For better compatibility and maintainability, it’s generally recommended to use static SVG files served over HTTP, inline <svg> elements (without embedded scripts or styles), or SVG sprites.

Reply:
I moved svg to js and html files respectively